test/fixtures/styled-components-nesting.js

```diff
@@ -0,0 +1,10 @@
+import styled, { css } from 'styled-components';
+const Message = styled.p`
+	padding: 10px;
+	${(props) => css`
+		color: #b02d00;
+	`}
+	${(props2) => css`
```

Comment:
I noticed that in every test there is `css` function before nested template literal. Will this work without it? E. g.

```js
import styled, { css } from 'styled-components';
const Message = styled.p`
	padding: 10px;
	${(props) => `
		color: #b02d00;
	`}
	${(props2) => `
		border-color: red;
	`}
`;
```

Could you add a test for this, please?

Reply:
Hi @hudochenkov.

I have added your requested test case.
https://github.com/stylelint/postcss-css-in-js/pull/17/files#diff-afcadaa7a1aaa2ea2d86507936d47fba

But this case doesn't parse the template literal inside as CSS. The inside template literal is converted to a `literal` node with no child nodes.

`TaggedTemplateExpression` is required to parse as CSS.

Reply:
Thank you!

I think it's totally ok, because it impossible to know what inside template literal without the tag.